packages/css/src/page-menu/page-menu.scss

```diff
@@ -69,6 +69,7 @@
 .amsterdam-page-menu__link:hover,
 .amsterdam-page-menu__button:hover {
   color: var(--amsterdam-page-menu-item-hover-color);
```

Comment:
I don't think this is necessary anymore, hover doesn't change the color by default

Reply:
I thought we could leave it in for other organisations wanting to use our components.